src/pages/tabs/tabs.ts

```diff
@@ -121,6 +121,7 @@ export class TabsPage {
 
     await this.checkCardEnabled();
     await this.tabProvider.prefetchCards();
+    this.checkAltCurrency(); // Check if the alternative currency setted is no longer supported
```

Comment:
I'm not completely sure about this line. tabPage is loaded first, then homePage. So, if those `await` don't exist, the action won't be called due to the event from home was not subscribed.

We should avoid that async issue to mitigate any random behavior when app is loading.